### ISSUE TITLE
Handle CDASH_CONFIG environment variable 

### ISSUE THREAD

**Issue by josephsnyder** [Kitware/CDash#1336]
Existing installations of CDash may have used the `CDASH_CONFIG`  environment variable in the docker-compose system to set local configuration values.  The old syntax of the environment variables:

```
        $$CDASH_USE_HTTPS = true;
        $$CDASH_SERVER_NAME = "cdash.ourdomain.com";
        $$CDASH_BASE_URL = "https://cdash.ourdomain.com";
        $$CDASH_ASYNCHRONOUS_SUBMISSION = true;
        $$CDASH_ASYNC_WORKERS = 4;
        $$CDASH_EMAILADMIN = "my.email@ourdomain.com";
        $$CDASH_EMAIL_FROM = "noreply@ourdomain.com";
        $$CDASH_EMAIL_REPLY = "noreply@ourdomain.com";
        $$CDASH_EMAIL_SMTP_HOST = "<IP of our SMTP server>";
        $$CDASH_EMAIL_SMTP_LOGIN = "noreply";
        $$CDASH_EMAIL_SMTP_PASS = "";
```

now no longer parses correctly when added to the `.env` file instead of `config.local.php`.   Change the place in `docker/bash/misc.bash` where the value is read and echoed to the file to ensure that the old formatting is properly dropped.

**Comment by josephsnyder** [comment]
Additional information and first report here: https://github.com/Kitware/CDash/pull/1293#discussion_r1151102397

**Comment by josephsnyder** [comment]
Closing as fixed by #1337.
